Doc changes from QA review (#5514)
cloudflare/cloudflare-docs: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/parameters.md

**Value:**

- `false` &mdash; (default) The user is able to turn the switch on/off at their discretion. When the switch is off, the user will not have the ability to reach sites protected by Access that leverage certain device posture checks.

> - `true` &mdash; The user is prevented from turning off the switch. The WARP Agent will start in the connected state with this is set to true.

Reviewer: ```suggestion
- `true` &mdash; The user is prevented from turning off the switch. The WARP client will start in the connected state.
```

### `auto_connect`

> If switch has been turned off by user, the client will automatically turn itself back on after the specified number of minutes. We recommend keeping this set to a very low value &mdash; usually just enough time for a user to log in to hotel or airport WiFi. If any value is specified for `auto_connect` the default state the app will always be Connected (ex. after reboot, after initial install, etc.)

Reviewer: ```suggestion
If switch has been turned off by user, the client will automatically turn itself back on after the specified number of minutes. We recommend keeping this set to a very low value &mdash; usually just enough time for a user to log in to hotel or airport WiFi. If any value is specified for `auto_connect` the default state of the WARP client will always be Connected (for example, after the initial install or a reboot).
```
